.k8s/10-zalando-postgres-ha-operator/README.md

```diff
@@ -0,0 +1,196 @@
+# Installing scalable PostgreSQL via Kubernetes operator
+
+## Prerequisites
+
+- See that there is no container named 'k3d-demo-tools'
+
+```
+docker ps -a
+```
+
+- Add two nodes (agents) to the cluster
+
+```
+k3d node create demo-agent -c demo --replicas 2
+```
+
+It's not an error: ```docker failed to remove the container 'k3d-demo-tools'```
+
+
+- Get the cluster nodes
+
+```
+k3d node list
+kubectl get nodes
+```
+
+## Steps
+
+
+### 1. Install ```postgres-operator``` via Helm Chart
+
+```
+helm repo add postgres-operator-charts https://opensource.zalando.com/postgres-operator/charts/postgres-operator
+helm repo update
+helm upgrade --install --cleanup-on-fail postgres-operator postgres-operator-charts/postgres-operator --namespace example-api --create-namespace
+```
+
+The operator and all its resources will be installed to namespace ```example-api```, which will be created if necessary
+
+
+### 2. See new Custom Resource Definitions 
+
+**From OpenLens**
+
+Custom Resources / acid.zalan.do / postgresql
+
+
+### 3. Create new PostgreSQL cluster
+
+**From CLI**
+
+```
+kubectl ns example-api
+cd ~/springboot-api-rest-example/.k8s/10-zalando-postgres-ha-operator
+kubectl apply -f db-crd.yaml
+```
+
+
+### 4. Watch how PostgreSQL cluster is creating in OpenLens 
+
+**From OpenLens**
+
+- Workloads / StatefulSets
+- Custom Resources / acid.zalan.do / postgresql
```

Comment:
kubectl -n ... get statefulsets
kubectl -n ... get crds
kubectl -n ... describe crd ...

Reply:
I use 

kubectl -n example-api get pods
kubectl -n example-api get pvc

instead of non-informative

kubectl -n ... get crds
kubectl -n ... describe crd ...